src/components/Options.js

```diff
@@ -1,10 +1,22 @@
 import React from "react"
 import Option from "./Option"
+import Radium from "radium"
 const Options = (props)=>{
 
+    const style = {
```

Comment:
Yeah, so this inline style thing.. It hurts the brain to look at when reading code.

You want to make your code much more readable by other developers and your future self. :)

The reason we abstract JSX out into other components is to keep our .js files small and semantic. It should be immediately clear what this component is about the second you open this file. Right now I open the file and my declaration of Options is way up by my imports... So it's not immediately clear that that's not an import. And then the first code I see is styles.. So I begin confused, is this a file that's about styles? And then I scroll down and see my return  and jsx and I realize, oh, this is a component who's just is to render a list of options.